api/graphite.go

```diff
@@ -831,6 +831,9 @@ type consolidatorTuple struct {
 // we will collect all the individual series from the peer, and then sum here. that could be optimized
 func (s *Server) executePlan(ctx context.Context, orgId uint32, plan *expr.Plan) ([]models.Series, models.RenderMeta, error) {
 	var meta models.RenderMeta
+	traceID, _ := tracing.ExtractTraceID(ctx)
+
+	log.Infof("executePlan: Starting traceID=%s req %v", traceID, plan.Reqs)
```

Comment:
It's worth noting this just dumps the patterns/`seriesByTag` expressions of the plan, not the whole function chain. Might be better to add somethign to `plan` (similar to dump() but maybe more tailored to this use case?)

Reply:
Plan is only aware of the individual `expr.expr`, the `expr.Req` and the `expr.GraphiteFunc`
If you wanted to log the original actual request (with the target expressions in string form), the plan would need to be modified to have that as a field. But actually, the plan was never designed to have awareness of that. it is simply the execution plan itself.  Why not log this message from `renderMetrics` instead ?

Reply:
@shanson7 ping ^

Reply:
I can look into that.